Client/Frontend/Browser/TabManager.swift

```diff
@@ -985,6 +985,14 @@ class TabManagerNavDelegate: NSObject, WKNavigationDelegate {
                 })
             }
 
+            if #available(iOS 11, *), res == .allow,
+                let appDelegate = UIApplication.shared.delegate as? AppDelegate,
```

Comment:
I'm not a fan of this. reaching into the UIApplication to pullout tabmanager feels very much an abuse of globals. Is there another way we can approach this?

Reply:
`TabManager.shared` ? I have awlays thought: it is a singleton, why can't I access it like one